recipe/build.sh

```diff
@@ -1,10 +1,17 @@
+# Get an updated config.sub and config.guess
+cp $BUILD_PREFIX/share/gnuconfig/config.* ./build
+cp $BUILD_PREFIX/share/gnuconfig/config.* ./contrib/ldapc++
 set -x
 # export CPPFLAGS="${CPPFLAGS} -isystem $PREFIX/include "
 
 # disable server components
 ./configure \
     --prefix=$PREFIX  \
     --disable-slapd \
-    --disable-slurpd || { cat config.log; exit 1; }
+    --disable-slurpd \
+    --with-yielding_select=yes || { cat config.log; exit 1; }
+
+sed -i.bak 's|#define NEED_MEMCMP_REPLACEMENT 1|/* #undef NEED_MEMCMP_REPLACEMENT */|' include/portable.h
```

Comment:
I am not sure this is the right solution, but can't find anything else that's working on google.

Reply:
You can do `export ac_cv_func_memcmp_working=yes`

Reply:
Done.

Reply:
You think this is ready to go?